### ISSUE TITLE
yolo v3

### ISSUE THREAD

**Issue by ssetty** [AlexeyAB/darknet#1303]
(1)
I need to plot loss functions is it possible to achieve this  using weights/checkpoints (for instance accuracy - epochs vs loss) - in this case this will be avg loss or avg IOU. Plots similar to tensorboard summary.

(2)
In your performance improvement guidelines you have mentioned about negative samples...
"desirable that your training dataset include images with non-labeled objects that you do not want to detect - negative samples without bounded box (empty .txt files) - use as many images of negative samples as there are images with objects"

Is below correct
1 0.0 0.0 0.0 0.0

That means if I have for class '1' 100 positive samples (without zeros) then I need to have for class '1' 100 negative samples ( with zeros) is this correct ?

Thanks
